readthedocs/templates/base.html

```diff
@@ -102,6 +102,11 @@
             <ul class="notifications">
               {% for message in messages %}
                 <li class="notification notification-{{ message.level }}" {% if message.pk %}data-dismiss-url="{% url 'message_mark_read' message.pk %}{% endif %}">
+                  {% if message.pk %}
+                  <a class="notification-action" href="{% url 'message_mark_read' message.pk %}">
+                    <span class="icon close" aria-label="Close notification"></span>
```

Comment:
Alternatively, instead of dropping the icon on both community/commercial, this span could default to text (`<span>Close</span>`), and on community the span would have text content removed via CSS and replaced with a FA icon

Reply:
Also, needs `{% trans %}` on text